unikube.py

```diff
@@ -21,6 +22,7 @@
 @click.group()
 @click.version_option()
 @click.pass_context
+@compare_decorator
```

Comment:
I don't think this is a good idea as the decorator is executed every time a cli is used. As discussed in #58 I would suggest to override the `message` parameter in the `version_option` decorator.

Something like this:

```python
def compare_current_and_latest_versions(): -> str
   # checking logic
   return version_info_with_or_without_comparison
   
@click.version_option(message=f"compare_current_and_latest_versions()")
def cli(ctx, **kwargs):
    ctx.obj = ClickContext()
    ...
```
However I'm not sure if this solution evaluates the string only when the cli is started with the `--version` flag, I at least hope so.

Reply:
I have tried the method you suggested and as you were suspecting, function is run every time, when I execute any `unikube` command. Moreover, version_option decorator is always executed, when any of the `unikube` commands is executed.

Reply:
Alright. So probably we should implement a dedicated command `version` and remove the decorator for `--version`?